src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -857,9 +857,9 @@ private boolean checkAtLeastTwoParticipants()
      * @return <tt>true</tt> if given {@link ChatRoomMember} is a focus
      *         participant.
      */
-    boolean isFocusMember(ChatRoomMember member)
+    boolean isFocusMember(XmppChatMember member)
     {
-        return member.getName().equals(focusUserName);
+        return focusUserName.equals(member.getJabberID().getResourceOrEmpty());
```

Comment:
If I am reading the code correctly, member.getName() is the resource part of the MUC JID, while getJabberID() returns the original JID (not the one in the MUC).